webview/src/plots/components/ribbon/Errors.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styles from './styles.module.scss'
+import { PlotsState } from '../../store'
+import { Error } from '../../../shared/components/icons'
+import { setShowErrorsModal } from '../webviewSlice'
+
+export const Errors: React.FC = () => {
+  const dispatch = useDispatch()
+  const [errorsTotal, setErrorsTotal] = useState(0)
+  const errors = useSelector((state: PlotsState) => state.webview.plotErrors)
+
+  useEffect(() => {
+    let total = 0
+    for (const { revs } of errors) {
+      total += revs.length
+    }
+    setErrorsTotal(total)
+  }, [errors])
+
+  if (errors.length === 0) {
+    return
+  }
+
+  return (
+    <li className={styles.errors}>
+      <p className={styles.errorsTitle}>
+        <Error className={styles.errorsIcon} width="16" height="16" />
+        Errors Found
+      </p>
+      <button
+        className={styles.errorsButton}
+        onClick={() => dispatch(setShowErrorsModal(true))}
+      >
+        {errorsTotal === 1 ? 'Show error' : `Show ${errorsTotal} errors`}
```

Comment:
Is there a text we could use for both single and plural errors? Maybe `Show` or `Show More` would be better?